### ISSUE TITLE
Indexer Request: 1337x

### ISSUE THREAD

**Issue by sp0rkeh** [cardigann/cardigann#87]
1337x (Public)
URL: http://1337x.to/
Language: English

I am regularly finding television content on 1337x that I can't find on other public trackers and even BTN. I would greatly appreciate an indexer definition file for this site.

As a side note, I couldn't find any documentation for creating indexer definition files. I would be willing to make this file myself (and submit it as a PR for everybody's benefit) if I understood how the definitions worked. Is there some documentation that I'm not seeing? If not, I'll try looking at the definition files some more and see if I can fully understand them. They look straightforward enough but it's just a headache trying to parse all of an unfamiliar format without any help. Are there any tips for writing these custom definitions? 

Thanks!

**Comment by lxt2** [comment]
Thanks @sp0rkeh, the format is stabilizing at the minute and then I will document it. See https://github.com/cardigann/cardigann/issues/5 for when that happens. 

In the mean time, if you wanted to copy an existing definition and ask some questions, I'd be happy to assist.

**Comment by sp0rkeh** [comment]
I just got around to attempting this and I don't think it's possible with cardigann. The site has a very strange design which causes a couple problems when making a definition file. The main issue I see is that there is no download button for torrents on the search results page, nor is there any way to calculate a download link from the information available on the search results page.

Unless there's a way to make cardigann navigate to the details page and then parse the download link from there, this seems like a dead-end. I figured I'd check just to make sure before giving up on this particular site. I love cardigann though, now that I've had a chance to mess around with it a little.
